src/main/java/fr/foodlimit/api/food/FoodController.java

```diff
@@ -16,7 +14,7 @@
  */
 @RestController
 @CrossOrigin
-@RequestMapping("/foods")
+@RequestMapping("/places/{placeId}/foods")
 public class FoodController {
 
   private final TokenProvider tokenProvider;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "tokenProvider" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)